### PR TITLE
#FRONTEND-71 - When the user opens the app it will show search view a…

### PR DESCRIPTION
**Pull request by osvold**
…s default.

### CODE REVIEW

File: app/src/main/java/io/hobaskos/event/eventapp/ui/main/MainActivity.java [hunk 1]
```diff
@@ -64,15 +64,15 @@ protected void onCreate(Bundle savedInstanceState) {
         App.getInst().getComponent().inject(this);
         setContentView(R.layout.activity_main);
 
-        if (googleServicesAvailable()) {
-            Toast.makeText(this, "Google Play Services er på plass!", Toast.LENGTH_LONG).show();
+        if(!googleServicesAvailable())
+        {
```
Comment (plastboks) [inline app/src/main/java/io/hobaskos/event/eventapp/ui/main/MainActivity.java]: Linjeskift :-1:

**Review by plastboks — APPROVED** [review]
Smooth

File: app/src/main/java/io/hobaskos/event/eventapp/ui/main/MainActivity.java (hunk 1 above)
Comment (osvold) [reply app/src/main/java/io/hobaskos/event/eventapp/ui/main/MainActivity.java]: Ops

**Comment by Barnholt** [comment]
Perf. 👌

**Pull request merged**